src/Features/Core/Portable/SolutionCrawler/IDocumentTrackingService.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Looks like IDocumentTrackingService.TryGetActiveDocument() should return nullable,